classes/Parameters/ConfigurationValidator.php

```diff
@@ -75,7 +75,7 @@ public function validate(array $array = []): array
 
     private function validateChannel(string $channel): ?string
     {
-        if ($channel !== UpgradeConfiguration::CHANNEL_LOCAL && $channel !== UpgradeConfiguration::CHANNEL_ONLINE) {
+        if ($channel !== UpgradeConfiguration::CHANNEL_LOCAL && $channel !== UpgradeConfiguration::CHANNEL_ONLINE && $channel !== UpgradeConfiguration::CHANNEL_ONLINE_RECOMMENDED) {
```

Comment:
```suggestion
        if (!in_array($channel, [
    UpgradeConfiguration::CHANNEL_LOCAL,
    UpgradeConfiguration::CHANNEL_ONLINE,
    UpgradeConfiguration::CHANNEL_ONLINE_RECOMMENDED,
], true)) {
```
what do you think about that ?